src/main/java/com/fyle/bank/models/BankBranches.java

```diff
@@ -0,0 +1,21 @@
+package com.fyle.bank.models;
```

Comment:
why do you have this model when each branch belongs only to one bank

Reply:
That package dude